<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedit_client_port">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>182</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>181</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>客户端IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lindit_client_addr">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>182</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>181</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>客户端Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>181</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lindit_serv_addr"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="linedit_server_port"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_bind">
      <property name="text">
       <string>绑定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
